crates/typst-library/src/text/deco.rs

```diff
@@ -67,7 +67,6 @@ pub struct UnderlineElem {
     /// #underline(background: true)[This is stylized.] \
     /// #underline(background: false)[This is partially hidden.]
     /// ```
-    #[default(false)]
```

Comment:
This removal is probably influenced by the quotes PR, but for boolean I actually prefer to have them explicit. The implicit false is not as obvious as an implicit auto or none.

Reply:
Sure thing!